features/issue.feature

```diff
@@ -80,6 +80,34 @@ Feature: hub issue
     """
     When I successfully run `hub issue -d 2016-08-18T09:11:32Z`
 
+  Scenario: Fetch issues
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :sort => "comments"
+      assert :direction => "asc"
+
+      json [
+        { :number => 13,
+          :title => "Second issue",
+          :state => "open",
+          :user => { :login => "octocat" },
+        },
+        { :number => 102,
+          :title => "First issue",
+          :state => "open",
+          :user => { :login => "octocat" },
+        },
+      ]
+    }
+    """
+    When I successfully run `hub issue -o comments -^`
+    Then the output should contain exactly:
```

Comment:
You don't have to assert any output here. The `When I successfully run…` line already verified that the sort critera was passed correctly. Otherwise, the fake API server would have returned a HTTP 400, and the command would have failed.